engines/groovie/groovie.cpp

```diff
@@ -146,22 +150,35 @@ Common::Error GroovieEngine::run() {
 		_grvCursorMan = new GrvCursorMan_t7g(_system, _macResFork);
 		_videoPlayer = new VDXPlayer(this);
 		break;
-	case kGroovieV2:
+
+	case kGroovieT11H:
+	case kGroovieCDY:
+	case kGroovieUHP:
+	case kGroovieTLC:
+		_font = FontMan.getFontByUsage(Graphics::FontManager::kBigGUIFont);
 		_resMan = new ResMan_v2();
 		_grvCursorMan = new GrvCursorMan_v2(_system);
 #ifdef ENABLE_GROOVIE2
 		_videoPlayer = new ROQPlayer(this);
 #endif
 		break;
+
 	default:
-		break;
+		error("GROOVIE: Unknown Game version. groovie.cpp:run()");
 	}
 
-	// Detect ScummVM Music Enhancement Project presence (T7G only)
-	if (Common::File::exists("gu16.ogg") && _gameDescription->version == kGroovieT7G) {
-		// Load player for external files
-		_musicPlayer = new MusicPlayerIOS(this);
-	} else {
+
+	switch (_gameDescription->version) {
+	case kGroovieT7G:
+		// Detect ScummVM Music Enhancement Project presence (T7G only)
+		if (Common::File::exists("gu16.ogg")) {
+			// Load player for external files
+			_musicPlayer = new MusicPlayerIOS(this);
+			break;
+		}
```

Comment:
according to the guidelines, I need to add a comment like `// fall through` here, however I was thinking it would be more clear if it said `// else, fall through`

Reply:
For currently known issues, you can use the groovie page in our wiki (or the individual game pages)